src/auditwheel/audit.rs

```diff
@@ -238,29 +238,30 @@ fn policy_is_satisfied(
         [lib] if is_libpython.is_match(lib) => {
             Err(AuditWheelError::LinksLibPythonError(lib.clone()))
         }
-        offenders => Err(AuditWheelError::ManylinuxValidationError(
+        offenders => Err(AuditWheelError::PlatformTagValidationError(
             policy.clone(),
             offenders.to_vec(),
         )),
     }
 }
 
 /// An reimplementation of auditwheel, which checks elf files for
-/// manylinux compliance.
+/// manylinux/musllinux compliance.
 ///
-/// If `manylinux`, is None, it returns the the highest matching manylinux policy, or `linux`
+/// If `platform_tag`, is None, it returns the the highest matching manylinux/musllinux policy, or `linux`
 /// if nothing else matches. It will error for bogus cases, e.g. if libpython is linked.
 ///
-/// If a specific manylinux version is given, compliance is checked and a warning printed if
+/// If a specific manylinux/musllinux version is given, compliance is checked and a warning printed if
 /// a higher version would be possible.
 ///
-/// Does nothing for manylinux set to off or non-linux platforms.  
+/// Does nothing for `platform_tag` set to `Off`/`Linux` or non-linux platforms.
 pub fn auditwheel_rs(
     path: &Path,
     target: &Target,
-    manylinux: Option<Manylinux>,
+    platform_tag: Option<PlatformTag>,
 ) -> Result<Policy, AuditWheelError> {
-    if !target.is_linux() || manylinux == Some(Manylinux::Off) {
+    // TODO: add support for musllinux: https://github.com/pypa/auditwheel/issues/305
```

Comment:
I think it's better to wait for upstream auditwheel to have a clear policy for musllinux tags before merging.

Reply:
With https://github.com/PyO3/maturin/pull/543/commits/4ed37993359b6f9dc4e9c96a0915e1c60da189c2 , I think it's ok to merge this without waiting for upstream.